### PR TITLE
Email integration

### PR DESCRIPTION
**Pull request by rhKashif**
- Integrated live data to email report
- Removed charts from report
- Added tables for: new releases, top releases by sentiment, top releases by number of reviews

### CODE REVIEW

**Review by DMcCallion — APPROVED** [review]
Looks good to me

File: report/main.py [hunk 1]
```diff
@@ -48,12 +49,54 @@ def get_database(conn_postgres: connection) -> DataFrame:
     Returns:
         DataFrame:  A pandas DataFrame containing all relevant release data
     """
-    query = f""
+    query = f"SELECT\
```
Comment (gawsalya) [inline report/main.py]: how chunky

File: report/main.py [hunk 2]
```diff
@@ -313,36 +486,27 @@ def create_report(df_releases: DataFrame) -> None:
...
-        </div>
-        <div class = "myDiv">
-            <p>{trending_game_two}</p>
-        </div>
-        <div class = "myDiv">
-            <p>{trending_game_three}</p>
-        </div>
     </body>    
     </html>
     '''
```
Comment (gawsalya) [inline report/main.py:512]: for your email -- put it in dotenv so people can change it easier and also it isn't leaked

**Review by gawsalya — APPROVED** [review]
good work!

File: report/main.py (hunk 1 above)
Comment (angelikavilde) [reply report/main.py]: does not need to be an f string

**Pull request merged**